yang/frr-gmp.yang

```diff
@@ -55,31 +55,32 @@ module frr-igmp {
      (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
      OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.";
 
-  revision 2019-11-06 {
+  revision 2021-11-22 {
     description
       "Initial revision.";
     reference
       "RFC 2236: IGMP v2.
-       RFC 3376: IGMP v3.";
+       RFC 3376: IGMP v3.
+       RFC 2710: MLD.";
   }
 
   grouping interface-config-attributes {
     description
-      "Configuration attributes applied to the interface level.";
+      "Configuration IGMP/MLD attributes applied to the interface level.";
 
-    leaf igmp-enable {
+    leaf enable {
       type boolean;
       default "false";
       description
-        "Enable IGMP flag on the interface.";
+        "Enable IGMP/MLD flag on the interface.";
     }
 
     leaf version {
       type uint8 {
         range "2..3";
```

Comment:
This is wrong for MLD. How will this be handled?

Reply:
Thanks for this point. I have resolved by creating igmp-version and mld version for V4 and V6 address-family separately.